py/desispec/ccdcalib.py

```diff
@@ -46,35 +48,62 @@ def compute_dark_file(rawfiles, outfile, camera, bias=None, nocosmic=False,
     Options:
         bias (str or list): bias file to use, or list of bias files
         nocosmic (bool): use medians instead of cosmic identification
-        scale (bool): apply scale correction for EM0 teststand data
         exptime (float): write EXPTIME header keyword; all inputs must match
+        min_vccdsec (float) : minimal time (in sec) after CCD bias voltage was turned on
+        max_temperature_diff (float) : maximal CCD temperature difference
+        reference_header (dict) : reference header that defines the valid hardware configuration (default is the most recent one)
+        save_preproc (bool) : save preprocessed images
+        output_specprod_dir (str) : specify output specprod directory used to save preproc images
+
+
 
-    Note: if bias is None, no bias correction is applied.  If it is a single
-    file, then use that bias for all darks.  If it is a list, it must have
-    len(rawfiles) and gives the per-file bias to use.
+    Note: if bias is None, the bias will be looked for in
+    $DESI_SPECTRO_REDUX/$SPECPROD/calibnight then $DESI_SPECTRO_CALIB
```

Comment:
I don't like this default, since in most cases that is the wrong bias to use and will result in bad preprocessing for the darks.  I would prefer the default to crash if it can't find the nightly biases it needs, with an option to opt-in to using a potentially dangerously bad bias as a fallback.  At minimum there should be an option to opt-out of using that fallback so that a missing nightly bias doesn't just trigger a log warning and move on.

Reply:
I changed the docstring. The current version of the code already raises an error if no biasnight is found.